apps/admin-ui/src/spa/application-rounds/[id]/review/ApplicationEventsTable.tsx

```diff
@@ -127,7 +156,15 @@ const COLS = [
     headerTKey: "ApplicationEvent.headings.phase",
     isSortable: true,
     key: "status",
-    transform: ({ statusView }: ApplicationEventView) => statusView,
+    transform: ({ status }: { status: ApplicationSectionStatusChoice }) => {
+      const { t } = i18n;
```

Comment:
use the `useTranslation` hook

Reply:
Can't be used here, as `getColConfig` isn't inside a component.. but I refactored it so we don't have a `COLS` anymore, as discussed.